drivers/crypto/Kconfig.stm32

```diff
@@ -0,0 +1,24 @@
+# STM32 Cryptographic Accelerator configuration options
+
+# Copyright (c) 2020 Markus Fuchs <markus.fuchs@de.sauter-bc.com>
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig CRYPTO_STM32
+	bool "STM32 Cryptographic Accelerator driver"
+	depends on SOC_FAMILY_STM32
+	select USE_STM32_HAL_CRYP
+	select USE_STM32_HAL_CRYP_EX
+	help
+	  Enable STM32 HAL-based Cryptographic Accelerator driver.
+
+if CRYPTO_STM32
+
+config CRYPTO_STM32_MAX_SESSION
+	int "Maximum of sessions STM32 crypto driver can handle"
+	default 2
+	depends on CRYPTO_STM32
```

Comment:
This line has no effect since redundant with "if CRYPTO_STM32"

Reply:
This line has been removed.